apps/advisor/kinds/check.cue

```diff
@@ -1,15 +1,21 @@
 package advisor
 
 check: {
-	kind:	   "Check"
+	kind:       "Check"
 	pluralName: "Checks"
-	current:	"v0alpha1"
+	current:    "v0alpha1"
+
+	codegen: {
+		go: {
+			enabled: true
+		}
+		ts: {
+			enabled: false
```

Comment:
Looks like frontend codegen was previously enabled in this kind, was there a reason this was changed to `false`?